File: modules/widgets/lib/aerox/widgets/event/CursorMoveEvent.cpp
#include "aerox/widgets/event/CursorMoveEvent.hpp"

namespace aerox::widgets
{
    CursorMoveEvent::CursorMoveEvent(const Shared<Surface>& inSurface, const Vec2<float>& inPosition) : Event(inSurface), position(inPosition)
    {

    }
}

File: aerox.Runtime.Widgets/Events/CursorMoveEvent.cs
using aerox.Runtime.Math;
using aerox.Runtime.Widgets.Graphics;

namespace aerox.Runtime.Widgets.Events;

public class CursorMoveEvent : Event
{
    public Vector2<float> Position;

    public CursorMoveEvent(Surface surface, Vector2<float> position) : base(surface)
    {
        Position = position;
    }
}